ibm/service/cis/resource_ibm_cis_rulesets_version_detach_test.go

```diff
@@ -0,0 +1,35 @@
+// Copyright IBM Corp. 2017, 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+package cis_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMCisRulesetsVersion_Basic(t *testing.T) {
+	name := "data.ibm_cis_rulesets_version_detach.test"
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheckCis(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCisAlertsDataSource_basic("test", acc.CisDomainStatic),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(name, "id"),
```

Comment:
Check for successful run of the function and not needed to compare attibutes. May be check ID